setup.py

```diff
@@ -1,4 +1,15 @@
-from setuptools import setup, find_packages
+from setuptools import setup
+import os
+
+
+def find_stubs(package):
```

Comment:
Just out of curiosity, will this sort of helper function be generally recommended for writing stub packages?

Reply:
Probably. I was thinking of making a change to either distutils or setuptools with helpers.